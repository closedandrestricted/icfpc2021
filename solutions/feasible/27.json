{"vertices":[[0,6],[28,2],[28,2],[7,21]]}