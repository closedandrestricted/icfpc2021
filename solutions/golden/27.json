{"vertices":[[0,0],[21,19],[15,24],[0,0]]}